Subproject2/3. Data Layer/AnswerRepository.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using _0._Models;
+using _3._Data_Layer.Database_Context;
+using Microsoft.EntityFrameworkCore;
+using System.Linq;
+
+
+namespace _3._Data_Layer
+{
+    public class AnswerRepository
+    {
+        private readonly SOVAContext databaseContext;
+
+        public AnswerRepository(SOVAContext databaseContext)
+        {
+            this.databaseContext = databaseContext;
+
+        }
+
+        public IEnumerable<Answer> GetLatestTenAnswers()
+        {
+            return databaseContext.Answers.OrderByDescending(s => s.Submission.CreationDate).Take(10);
+        }
```

Comment:
remove this as this method is not relevant here and push again. I'll merge it after that :)